glom/test/test_basic.py

```diff
@@ -4,14 +4,13 @@
 
 import pytest
 
-from glom import glom, SKIP, STOP, Path, Inspect, Coalesce, CoalesceError, Val, Call, T, S, Invoke, Spec, Ref
-from glom import Auto, Fill, Iter, A, Vars, Val, Literal, GlomError, Pipe
+from glom import glom, SKIP, STOP, Inspect, Coalesce, Val, Call, T, S, Invoke, Spec, Ref
+from glom import Fill, Iter, Literal, Pipe
```

Comment:
ah, these imports are to ensure we don't regress the __init__ of the module itself, I don't think we want to remove them.